packages/api-client/tailwind.config.ts

```diff
@@ -47,7 +47,7 @@ export default {
       },
       spacing: {
         /** Client headers including the address bar and the section headers */
-        'client-header': '52px',
+        'client-header': '48px',
```

Comment:
Are we still using this or did you remove that code?

Reply:
seems used so leaving this there for now on